.gitmodules

```diff
@@ -2,3 +2,6 @@
   path = flatland
   url = git@github.com:avidbots/flatland.git
   branch = ros2-humble
+[submodule "beluga/docs/doxygen-awesome-css"]
+  path = beluga/docs/doxygen-awesome-css
+  url = git@github.com:jothepro/doxygen-awesome-css.git
```

Comment:
I'm not a big fan of adding everything as a submodule.
For build dependencies I think it's better to use something like `vcs`.

In this case, we could pull the dependency in the CMakeLists that build the docs (with ExternalProject_Add or FetchContent).

Reply:
Done in https://github.com/Ekumen-OS/beluga/pull/204/commits/6a765db8bb518d6c6c3db6a07241b7778e4b3b3a.